src/helpers.js

```diff
@@ -80,8 +80,27 @@ var insertTask = function(namespace, input, options) {
   });
 };
 
+var getModelData = function({query, limit, continuation, key, Model}, callback) {
+  return Model.query(query, {
+    limit,
+    continuation,
+  }).then(function(data) {
```

Comment:
check if this repo doesn't support async... and then use arrow functions.

Reply:
async/await is available in this service (and in all TC services)